tests/TestOpVaultReader.cpp

```diff
@@ -52,7 +52,7 @@ QPair<QString, QString>* split1PTextExportKV(QByteArray& line)
 QJsonArray* read1PasswordTextExport(QFile& f)
 {
     auto result = new QJsonArray;
```

Comment:
This should be a QSharedPointer.

Reply:
I agree that this raw pointer isn't great, but I don't think it's a leak. I'm happy to change it though. I would think `QScopedPointer` makes more sense here though, since it doesn't need to be shared. The return type of `read1PasswordTextExport` would then become `QScopedPointer<QJsonArray>`.

Reply:
Ah I see you can't return a `QScopedPointer` from a function; that's a shame.

I've changed it to use `std::unique_ptr` now. Hope that's ok. I see it's not used a lot in keepassxc, but there are a few other places that use it.

Reply:
Reviewing previous discussions around this, I see Qt is preferred over std wherever possible, so changed to `QSharedPointer` now.

Reply:
Yes.